lib/ui/LandingPage.dart

```diff
@@ -213,8 +211,11 @@ class LandingPage extends StatelessWidget {
     BuildContext context,
     LandingPageViewModel viewModel,
   }) {
-    NavigationScope.presentModal(
-        context, ChatPage(viewModel: viewModel.chatViewModel));
+    viewModel.newAccountFlow.run(context, onSuccess: (){
+      print("YES");
```

Comment:
Should be localised string?

Reply:
I don't think so

Reply:
that's debug code, I should remove. this callback if only for if you want to do something on the view layer, which we don't need in this case ATM.

Reply:
sorted 👍